src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -2550,6 +2550,21 @@ public bool CanUnloadDatabase(StringSegment databaseName, DateTime lastRecentlyU
                     statistics.Explanations.Add($"Cannot unload database because number of Changes API connections ({numberOfChangesApiConnections}) is greater than 0");
                 }
             }
+            using (database.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var numberOfSubscriptionConnections = database.SubscriptionStorage.GetAllRunningSubscriptions(context, false, 0, int.MaxValue).ToList().Count;
```

Comment:
Create a dedicated method for that count. `GetNumberOfRunningSubscriptions()`

Reply:
done